src/test/java/executor/service/model/ScenarioTest.java

```diff
@@ -0,0 +1,82 @@
+package executor.service.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class ScenarioTest {
+
+    private Scenario scenario;
+
+
+    //Initialize a new script before each test
+    @Before
+    public void setUp() {
+        List<Step> steps = new ArrayList<>();
+        steps.add(new Step("Action 1", "Value 1"));
+        steps.add(new Step("Action 2", "Value 2"));
+
+        scenario = new Scenario("Test Scenario", "example.com", steps);
+    }
```

Comment:
Maybe better use static method _prepareScenario()_ instead?
No need for comment above
.